lib/components/SelectionControl/checkbox/checkbox-group/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { useState, useEffect } from "react";
+import clsx from "clsx";
+import "./checkbox-group.scss";
+import { Checkbox } from "../checkbox-single";
+import { CheckboxProps } from "../checkbox-single";
```

Comment:
I will merge this PR, but can we apply this change? 

```suggestion
import { Checkbox, CheckboxProps } from "../checkbox-single";
```